addons/hr_holidays/static/src/leave_stats/leave_stats.xml

```diff
@@ -10,7 +10,12 @@
             </div>
             <div t-foreach="state.leaves" t-as="leave" t-key="leave_index" class="d-flex flex-row justify-content-between">
                 <span t-esc="leave.holiday_status_id[1]"/>
-                <span><t t-esc="leave.number_of_days"/> day(s)</span>
+                <t t-if="leave.leave_type_request_unit === 'day'">
```

Comment:
Check if it is different than `hour` here, because it could be both 'day' and 'half_day'
```suggestion
                <t t-if="leave.leave_type_request_unit !== 'hour'">
```